website/src/app.jsx

```diff
@@ -11,6 +11,12 @@ export default React.createClass({ // eslint-disable-line
     return Object.assign({}, initialState)
   },
 
+  componentWillUpdate() {
+    const {refs: {maskedInput}} = this
+
+    maskedInput.setState(maskedInput.getInitialState())
```

Comment:
Yeah I know, just seems like a weird hack to reach into a component when we can store the data externally and `onChange` we'd change the external value to an empty string.